platforms/minigame/platforms/bytedance/wrapper/engine/VideoPlayer.js

```diff
@@ -338,7 +351,9 @@ if (cc.internal.VideoPlayer) {
         }
 
         syncMatrix () {
```

Comment:
Doing `uploadData` in syncMatrix method is really strange. Original code did the matrix synchronization in this function since we used the native controls.

Now, we use `VideoTexture`, but we still re-use the `syncMatrix` method. 
I think it should be renamed to `update` in another PR.

Reply:
This is a minimal modification, otherwise it might have been changed to a different platform.
This design may need to be modified subsequently